src/modules/pmc/tests/PmcTests.cpp

```diff
@@ -141,6 +159,36 @@ namespace OSConfig::Platform::Tests
         ASSERT_STREQ(reportedJsonPayload, payloadString.c_str());
     }
 
+    TEST_F(PmcTests, SetGetUpdatingPackagesSourcesFailure)
+    {
+        const std::map<std::string, std::tuple<int, std::string>> textResults =
+        {
+            {"command -v apt-get", std::tuple<int, std::string>(MMI_OK, "")},
+            {"command -v apt-cache", std::tuple<int, std::string>(MMI_OK, "")},
+            {"command -v dpkg-query", std::tuple<int, std::string>(MMI_OK, "")},
+            {"apt-get update", std::tuple<int, std::string>(EBUSY, "")},
+            {"dpkg-query --showformat='${Package} (=${Version})\n' --show | sha256sum | head -c 64", std::tuple<int, std::string>(MMI_OK, "25abefbfdb34fd48872dea4e2339f2a17e395196945c77a6c7098c203b87fca4")},
+            {"apt-cache policy cowsay | grep Installed", std::tuple<int, std::string>(MMI_OK, "  Installed: (none) ")},
+            {"apt-cache policy sl | grep Installed", std::tuple<int, std::string>(MMI_OK, "  Installed: (none) ")},
+            {"apt-cache policy bar | grep Installed", std::tuple<int, std::string>(MMI_OK, "  Installed: (none) ")},
+            {"find sources/ -type f -name '*.list' -exec cat {} \\; | sha256sum | head -c 64", std::tuple<int, std::string>(MMI_OK, "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b877")}
+        };
+        char reportedJsonPayload[] = "{\"packagesFingerprint\":\"25abefbfdb34fd48872dea4e2339f2a17e395196945c77a6c7098c203b87fca4\",\"packages\":[\"cowsay=(none)\",\"sl=(none)\",\"bar=(none)\"],\"executionState\":3,\"executionSubState\":6,\"executionSubStateDetails\":\"\",\"sourcesFingerprint\":\"e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b877\",\"sourcesFilenames\":[\"key.list\"]}";
```

Comment:
Break this super long string constant in substrings

"                              ";

->

"  "
"  "
"  ";